intl/msg_hash_ar.h

```diff
@@ -3438,7 +3566,7 @@ MSG_HASH(
    )
 MSG_HASH(
    MENU_ENUM_LABEL_VALUE_CHEEVOS_START_ACTIVE,
-   "شغل النشط"
+   "Encore Mode" /* suggestion for translators: translate as "Play Again Mode" */
```

Comment:
This. What is this? Why is this? Why can't I find this string on Crowdin? What is going on here!?

Reply:
Just to be clear, I know why the string was changed. What I don't understand is why it's in English _in all translations_ and, well, the second to last question from above.

Reply:
I kinda have the memory of, the last time Crowdin's import system broke up, was because of the appearance of additional quotation marks around comments. The GitHub->Crowdin process had issues with that.